dashboard/app/controllers/project_versions_controller.rb

```diff
@@ -24,7 +24,7 @@ def project_commits
     user_storage_id, project_id = storage_decrypt_channel_id(params[:channel_id])
     project_owner = User.find_by(id: user_id_for_storage_id(user_storage_id))
     return render :not_acceptable unless project_owner
-    return render :forbidden unless can?(:project_commits, ProjectVersion.new, project_owner, project_id)
+    return render :forbidden unless can?(:project_commits, ProjectVersion.new, project_owner)
```

Comment:
(minor) This seems a little strange to create a ProjectVersion object and then not use it at all.  Maybe define an action on Project or User instead?

Also, should the create endpoint above check some kind of permissions?  (ok to fix later).

Reply:
I'm not going to fix create endpoint right now